### ISSUE TITLE
CS0213 IntelliSense false positive (You cannot use the fixed statement to take the address of an already fixed expression)

### ISSUE THREAD

**Issue by NSentinel** [dotnet/roslyn#18079]
**Version Used**: 
Visual Studio 2017 RTM / 15.0.0 + 26228.9 (Also got this in VS 2015)

**Code to Reproduce**:

```csharp
internal unsafe struct Test
{
      public fixed byte Data[16];
}
```

```csharp
internal sealed class State
{
      internal Test Test;
}
```

**Actual Behavior**:

```csharp
fixed (byte* sTest = state.Test.Data) // CS0213
{
}
```
Code still compilable, error shown only in IntelliSense error list

**Comment by jcouv** [comment]
I'm having trouble to repro this. Could you try again on recent VS and let me know if you still see the issue?

I used the following code in a simple project and built. The Error List (Build + IntelliSense) shows no errors.
```c#
public unsafe struct Test
{
    public fixed byte Data[16];
}
public sealed class State
{
    public Test Test;

    unsafe void M(State state)
    {
        fixed (byte* sTest = state.Test.Data)
        {
        }
    }
}
```

**Comment by NSentinel** [comment]
Right now in Version 15.4.4 issue already gone so I'll close it. 

Thank you!

---
Microsoft Visual Studio Enterprise 2017
Version 15.4.4
VisualStudio.15.Release/15.4.4+27004.2009
Microsoft .NET Framework
Version 4.7.02558
